src/cascadia/TerminalSettingsEditor/AISettings.xaml

```diff
@@ -38,28 +38,36 @@
                 <TextBlock x:Uid="AISettings_ServiceProvidersHeader"
                            Margin="0,8,0,0"
                            Style="{StaticResource TextBlockSubHeaderStyle}" />
-                <!--  AIKey  -->
+                <!--  Azure OpenAI  -->

```

Comment:
Mind posting an image of the new UI here?

Reply:
![image](https://github.com/user-attachments/assets/ef9f69f3-5cc7-4a79-a9bf-eaad581d383f)